datasets/glossis/glossis.py

```diff
@@ -318,26 +351,24 @@
             )
         )
 
-
-        if not args.skip_flowmap_history:
-            # This should result in flowmap tiles in a bucket
-            # The flowmaps are tiled using a rather specific tile format
-            # These are  uploaded to the public bucket
-            # we're downloading some local files.
-            # do this using a temporary directory
-            # this will be cleaned up when we exit the context              
-            for i, row in work.iterrows():
-                flowmap_tiff = row.flowmap_tiff
-                logger.info("downloading {}".format(
-                    flowmap_tiff
-                ))
-                # write temp files to this directory
-                # change to this directory and return once we're done
-                with tempfile.TemporaryDirectory() as tmp_dir:
-                    with cd(tmp_dir):
-                        download_blob(flowmap_tiff)
-
-                        tile_dir = generate_wgs84_tiles(row.path)
-                        upload_dir_to_bucket(
-                            public_bucket, source_dir_name=tile_dir, destination_dir_name="flowmap/glossis/tiles"
-                        )
+        # This should result in flowmap tiles in a bucket
+        # The flowmaps are tiled using a rather specific tile format
+        # These are  uploaded to the public bucket
```

Comment:
These four comment lines are a bit vague instructions. Could be improved or removed.